kotlinx-coroutines-debug/src/CoroutinesBlockHoundIntegration.kt

```diff
@@ -10,6 +10,9 @@ import kotlinx.coroutines.scheduling.*
 import reactor.blockhound.*
 import reactor.blockhound.integration.*
 
+/**
+ * @suppress
```

Comment:
To get rid of this one: https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-debug/kotlinx.coroutines.debug/-coroutines-block-hound-integration/